### DISCUSSION TITLE
Will export rates 'pop up'?

### DISCUSSION THREAD

**Discussion by JRascagneres** [BottlecapDave/HomeAssistant-OctopusEnergy#515] [Q&A] [answered]
Having solar / battery installed soon so will be exporting. Will the integration make a couple new entities for export? Do I need to remove + re-add integration? etc.

**Comment by BottlecapDave** [comment]
Congrats! If you are getting new meters, then you'll just need to restart HA or reload the integration once they are assigned to your OE account.

**Reply by JRascagneres** [reply]
I think its the same meter, current meter should support it. Would it be the same? Just need to be on an export tarriff and it'll pick it up?

**Reply by BottlecapDave** [reply]
If you're on export, an additional meter will appear which will represent your export meter and tariff. You just need to reload the integration for it to appear.

**Comment by lamixer** [comment, marked as answer]
Mine just 'popped up' when they turned on my export tariff.

**Reply by JRascagneres** [reply]
Awesome thanks
